src/args.rs

```diff
@@ -783,7 +783,17 @@ impl<'a> ArgMatches<'a> {
         if let Some(limit) = self.regex_size_limit()? {
             gb = gb.size_limit(limit);
         }
-        gb.build().map_err(From::from)
+        gb.build().map_err(|err| {
+            match err {
+                GrepError::Regex(err) => {
+                  let s = format!("{}\nIf you wanted to treat the \
+                  string within double quotes as a literal string, you \
+                  should pass the --fixed-strings flag.", err.to_string());
```

Comment:
I think I would just write, `\n(Hint: try the --fixed-strings flag to search a literal.)`